config/hubs/schema.yaml

```diff
@@ -158,7 +158,7 @@ properties:
               - basehub
               - daskhub
           auth0:
-            additionalProperties: false
+            additionalProperties: true
```

Comment:
I needed to add this so that I could add `enabled: false` in a `*.cluster.yaml` file without experiencing a failure. It would also be nice to modify this schema in such a way that if `auth0.enabled: false` then the `auth0.connection` key is no longer required.